contrib/database/sql/option_test.go

```diff
@@ -8,8 +8,8 @@ package sql
 import (
 	"testing"
 
+	"github.com/stretchr/testify/assert"
```

Comment:
This import needs to go last and have an empty line between dd-trace-go imports and itself.